src/Http/composer.json

```diff
@@ -27,6 +27,7 @@
         "cakephp/core": "^4.0",
         "cakephp/event": "^4.0",
         "cakephp/utility": "^4.0",
+        "composer/ca-bundle": "^1.2",
```

Comment:
Sync version constraint with main `composer.json`

Reply:
I can update main file to `^1.2`

Reply:
Both variants are OK.

Reply:
Where is the prefer lowest run now on master? With travis this used to be a) visible and b) daily run using cronjobs. Now I cant find it anymore.

Reply:
removed when merging 3.x to 4.x

Reply:
Thats bad :/